local/nginx/include/lambdas.conf

```diff
@@ -1,4 +1,9 @@
 
+location ~ ^/lambdas/contentV2/(.*)$ {
```

Comment:
Why this way and not removing the endpoint completely?

Reply:
You are completely right